api/client/swarm/opts.go

```diff
@@ -18,6 +18,8 @@ const (
 	WORKER = "WORKER"
 	// MANAGER constant for manager name
 	MANAGER = "MANAGER"
+	// NONE constant
+	NONE = "NONE"
```

Comment:
They're being used in `opts_test.go`. So either they stay exported (but CamelCase) or we hard code the values into all the test cases?